radixdlt/src/main/java/com/radixdlt/CerberusModule.java

```diff
@@ -89,7 +89,7 @@ private EpochRx epochRx(
 		@Named("self") ECKeyPair selfKey,
 		AddressBook addressBook
 	) {
-		final int fixedNodeCount = Integer.parseInt(runtimeProperties.get("consensus.fixed_node_count", "1"));
+		final int fixedNodeCount = runtimeProperties.get("consensus.fixed_node_count", 1);

```

Comment:
Use integer get, rather than string get and conversion with no useful error message.